tests/hpc/enable_in_zypper.pm

```diff
@@ -18,8 +18,6 @@ use utils;
 use lockapi;
 
 sub run {
-    select_console 'root-console';
```

Comment:
not sure , there is no harm if console was selected before and by removing this line you break usage of enable_in_zypper anywhere else

Reply:
but where is enable_in_zypper called except HPC tests?

Reply:
ok , if someone will start using it , he can add this line back